src/CssToInlineStyles.php

```diff
@@ -33,19 +34,24 @@ public function __construct()
      */
     public function convert($html, $css = null)
     {
-        $document = $this->createDomDocumentFromHtml($html);
+        $document  = $this->createDomDocumentFromHtml($html);
         $processor = new Processor();
 
         // get all styles from the style-tags
-        $rules = $processor->getRules(
-            $processor->getCssFromStyleTags($html)
-        );
+        $styleTagCss = $processor->getCssFromStyleTags($html);
+
+        // proccess style tag css
+        $mediaQueries = $processor->getMediaQueries($styleTagCss);
```

Comment:
this does not make sense. style tags which are in the DOM are preserved in the output, so these media queries are already part of the output.

Reply:
btw, none of your new tests are covering that part.

Reply:
I see now, I'll revert this